ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -26,7 +26,7 @@ let prototyped_of_field = function
   | "Observer", "uuid" ->
       Some "23.14.0"
   | "Repository", "origin" ->
-      Some "24.21.0-next"
+      Some "24.23.0"
```

Comment:
Given that xapi is already on 24.24.0, would this version not need incrementing? Or am I misunderstanding things here?

Reply:
Would agree - needs a rebase? @robhoes

Reply:
Those calls went into 24.23.0, didn't they? 24.24.0 only had a revert commit.